pom.xml

```diff
@@ -1,4 +1,8 @@
-<project>
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
The xmlns attribute is defintiely a good idea, IMHO. The other two are at best optional.